### DISCUSSION TITLE
how can i use this project?

### DISCUSSION THREAD

**Discussion by digerati73** [kenjdavidson/react-native-bluetooth-classic#267] [Q&A]
this is first time to load react-native project.

i will use visual studio code.

i did several things to make settings for react native develop.

and i open folder at visual studio code.

and then?

first of all ,
i want to run this project and run ios app ,
so i want to ios app communicate with esp32 using bluetooth classic.

**Comment by kenjdavidson** [comment]
Not possible on ios with esp32.
